include/substrait-mlir/Dialect/Substrait/IR/SubstraitAttrs.td

```diff
@@ -121,12 +121,13 @@ def Substrait_FixedCharAttr
 }
 
 def Substrait_FixedBinaryAttr : Substrait_Attr<"FixedBinary", "fixed_binary",
-  [TypedAttrInterface ]> {
+    [TypedAttrInterface]> {
   let summary = "Substrait fixed-length binary type";
   let description = [{
     This type represents a substrait binary string of L bytes.
   }];
   let parameters = (ins "StringAttr":$value, "FixedBinaryType":$type);
+  let genVerifyDecl = 1;
   let assemblyFormat = "`<` custom<AddRemovePadding>($value, $type) `>`";
```

Comment:
Unfortunately, I didn't see that yet in our discussion today: like for `FixedChar`, the length is now redundant in the assembly -- the string literal's length already carries that information. Should we not remove the explicit length from the assembly format here as well? (I think that the benefit of not needing the length by not allowing padding weighs more than the benefit of shortening strings with trailing null characters, so I think this was a win-win choice!)

Also, `AddRemovePadding` doesn't fit anymore.